src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -48,14 +51,34 @@ private static bool IsInvalidDouble(double number)
                 Func<IRContext, int, FormulaValue, FormulaValue> checkRuntimeTypes,
                 Func<IRContext, int, FormulaValue, FormulaValue> checkRuntimeValues,
                 ReturnBehavior returnBehavior,
-                Func<EvalVisitor, EvalVisitorContext, IRContext, T[], ValueTask<FormulaValue>> targetFunction)
+                Func<EvalVisitor, EvalVisitorContext, IRContext, T[], ValueTask<FormulaValue>> targetFunction,
+                bool isMultiArgTabularOverload = false)
             where T : FormulaValue
         {
             return async (runner, context, irContext, args) =>
             {
                 var nonFiniteArgError = FiniteArgumentCheck(functionName, irContext, args);
                 if (nonFiniteArgError != null)
                 {
+                    (var maxSize, var minsSize, var emptyTablePresent, _, var errorTable) = AnalyzeTableArguments(args, irContext);
+
+                    // In case Tabular overload has one scalar error arg and another Table arg we want to
+                    // return table of error. else If error arg is table then return table error args.
+                    if (isMultiArgTabularOverload && maxSize > 0 && errorTable == null)
+                    {
+                        var tableType = (TableType)irContext.ResultType;
+                        var resultType = tableType.ToRecord();
+                        var namedValue = new NamedValue(tableType.SingleColumnFieldName, nonFiniteArgError);
+                        var record = DValue<RecordValue>.Of(new InMemoryRecordValue(IRContext.NotInSource(resultType), new List<NamedValue>() { namedValue }));
+                        var resultRows = Enumerable.Repeat(record, maxSize).ToList();
+
+                        return new InMemoryTableValue(irContext, resultRows);
+                    }
+                    else if (isMultiArgTabularOverload)
```

Comment:
>isMultiArgTabularOverload [](http://example.com/codeflow?start=29&length=25)

If `errorTable == null` and `maxSize == 0` we will return a null value from this method, which shouldn't be the case. Let's also add a test that would cover this case.